From 83388f12cca7984c0087e7bb31aaaf9d938a81f6 Mon Sep 17 00:00:00 2001
From: Krzysztof Kanas <kkanas@marvell.com>
Date: Mon, 8 Jul 2019 14:18:32 +0200
Subject: [PATCH 142/154] common/octeontx2: cleanup void returns

mbox_unregister_vf_irq and mbox_unregister_pf_irq returns void value.
mbox_unregister_irq also returns void.

Clang with flags '-Wall -Wextra -pedantic' complains about:
void function should not return void expression

Signed-off-by: Krzysztof Kanas <kkanas@marvell.com>
---
 drivers/common/octeontx2/otx2_dev.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/common/octeontx2/otx2_dev.c b/drivers/common/octeontx2/otx2_dev.c
index 53a0c6efbcbc..3b51fa93ad29 100644
--- a/drivers/common/octeontx2/otx2_dev.c
+++ b/drivers/common/octeontx2/otx2_dev.c
@@ -744,9 +744,9 @@ static void
 mbox_unregister_irq(struct rte_pci_device *pci_dev, struct otx2_dev *dev)
 {
 	if (otx2_dev_is_vf(dev))
-		return mbox_unregister_vf_irq(pci_dev, dev);
+		mbox_unregister_vf_irq(pci_dev, dev);
 	else
-		return mbox_unregister_pf_irq(pci_dev, dev);
+		mbox_unregister_pf_irq(pci_dev, dev);
 }
 
 static int
-- 
2.14.4

